tests/functional/test_custom_task_templates.py

```diff
@@ -435,7 +435,7 @@ def test_custom_task_templates(
                 "sklearn_drop_in_env",
                 "transform",
                 [
-                    "schema validation failed for input",
+                    "Schema validation found mismatch between input dataset and the supplied schema",
```

Comment:
The original error message is part of the stacktrace. Explicitly exit a program (e.g., equivalent with raising SystemExit) seems to change the stdout/stderr contents. Some content in stacktrace is truncated. This updates the expected content in the stacktrace accordingly.